master_changes.md

```diff
@@ -131,7 +131,10 @@ users)
 ## opam-format
 
 ## opam-core
-  * `OpamStd.Sys.{get_terminal_columns,uname,getconf,guess_shell_compat}`: Harden the process calls to account for failures [#6230 @kit-ty-kate - fix #6215]
-  * `OpamStd.Sys.{uname,getconf}`: now accepts only one argument as parameter, as per their documentation [#6230 @kit-ty-kate]
+  * `OpamStd.Sys.{get_terminal_columns,guess_shell_compat}`: Harden the process calls to account for failures [#6230 @kit-ty-kate - fix #6215]
+  * `OpamStd.Sys.getconf`: was removed [#6217 @kit-ty-kate]
+  * `OpamStd.Sys.uname`: now returns the memoized result of `uname(2)` [#6217 @kit-ty-kate]
```

Comment:
In some distros, the `uname` C function is `uname(3)`. I think its better to refer it in plain text.
```suggestion
  * `OpamStd.Sys.uname`: now returns the memoized result of `uname` C binding [#6217 @kit-ty-kate]
```